botchat.css

```diff
@@ -122,6 +116,11 @@
         position: relative;
     }
 
+    .wc-message-wrapper.carousel .wc-message {
```

Comment:
this is correct - it is selecting a wc-message inside of a wrapper with the carousel layout